images/Dockerfile.kairos

```diff
@@ -73,11 +74,8 @@ LABEL io.kairos.software-version="${SOFTWARE_VERSION}"
 LABEL io.kairos.software-version-prefix="${SOFTWARE_VERSION_PREFIX}"
 LABEL io.kairos.targetarch="${TARGETARCH}"
 
-# not duplicated but used to see the error
-RUN kairos-agent versioneer os-release-variables
 RUN kairos-agent versioneer os-release-variables >> /etc/os-release
-# not duplicated but used to see the error
-RUN kairos-agent versioneer container-artifact-name
+RUN kairos-agent versioneer os-release-variables > /etc/kairos-release
```

Comment:
why do we append to os-release too? (line 77)

Reply:
backwards compatibility? Im not sure if we should drop it directly. It makes sense to not write them, anymore in there from now on, but I dont know...

Reply:
yeah good question, I have the feeling that if we know that an upgrade can nuke this file, we should just move away from it at this point and already start seeing where it could break